pcdsdevices/mpod.py

```diff
@@ -181,3 +181,38 @@ def __init__(self, channel_prefix, card_prefix, name='mpod_hv_channel',
                  **kwargs):
         self._card_prefix = card_prefix
         super().__init__(channel_prefix, name=name, **kwargs)
+
+
+def MPOD(channel_prefix, card_prefix=None, **kwargs):
+    """
+    Determine the appropriate MPOD Channel class based on the channel number.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        Epics base PV for channels.
+    card_prefix : str
+        Epics base PV for the whole MPOD module, HV Channels.
+    kwargs : dict
+        Information to pass through to the device, upon initialization
+    """
+    # CH below 100 should be LV, and above or 100 should be HV.
+    # maybe also try to determine the card_prefix as well if none?
+    try:
+        channel = str(channel_prefix).split('CH:')[1]
+        base = str(channel_prefix).split('CH:')[0]
+        channel = int(channel)
+    except Exception:
+        # Default to ophyd.MPODChannel
+        logger.warning('Unable to figure out the type of mpod channel based on'
+                       ' channel number. Using "ophyd.MPODChannel"')
+        return None
```

Comment:
Warning says "using MPODChannel" but this returns `None`?

Reply:
Thank you, fixed it in 80f48ba. I initially wanted to return a `MPODChannel` - but it makes not sense to create one when we could not figure out the channel number i think